test/encoder/EncUT_SliceBufferReallocate.cpp

```diff
@@ -211,7 +211,7 @@ void CSliceBufferReallocatTest::InitParam() {
     pCtx->pSvcParam->iMultipleThreadIdc = (pCtx->pSvcParam->iMultipleThreadIdc <= 1 )? 2 : pCtx->pSvcParam->iMultipleThreadIdc;
     pCtx->iActiveThreadsNum = pCtx->pSvcParam->iMultipleThreadIdc;
     pCtx->pSvcParam->iSpatialLayerNum = 1;
-    pCtx->pSvcParam->bSimulcastAVC = (bool)rand() % 2;
```

Comment:
is this a new warning or a old one?
why the force type conversion does not work on MSVC? 
thanks.